includes/rest.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+// rest api functions
+
+add_action('rest_api_init', function () {
+    register_rest_route('tsml', '/meetings', [
+        'methods' => 'GET',
+        'callback' => 'tsml_rest_feed_endpoint',
```

Comment:
Left the version out of the path, probably won't have a ton of future endpoints or breaking changes to this one